README.md

```diff
@@ -18,7 +18,7 @@ To use SProcWrapper, add the following lines to your pom.xml:
 
 ```xml
 <dependency>
-    <groupId>de.zalando</groupId>
+    <groupId>org.zalando</groupId>
     <artifactId>zalando-sprocwrapper</artifactId>
```

Comment:
Should we rename it to `sprocwrapper`?

Reply:
We can change that as well. Github will manage a redirect, so we don't loose anything.

Reply:
@Jan-M Objections?

Reply:
i have no objections, given there is some rule of using .org for open source libraries.